crm_partner_assign/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+{
+    "name": "CRM Partner Assign",
+    "summary": """
+        Assign a Partner to an Opportunity/Lead/Partner to indicate Partnership,
+    """,
```

Comment:
Incorrect change. If it doesn't fit, you should split the string, but don't introduce linebreaks with `"""`.

```suggestion
    "summary": "Assign a Partner to an Opportunity/Lead/Partner"
    " to indicate Partnership",
    """,
```

Reply:
done

Reply:
This is still not done.